src/getThreadList.js

```diff
@@ -4,22 +4,28 @@ var utils = require("../utils");
 var log = require("npmlog");
 
 module.exports = function(defaultFuncs, api, ctx) {
-  return function getThreadList(start, end, callback) {
+  return function getThreadList(start, end, pending, callback) {
```

Comment:
You could insert this at line [#15](https://github.com/NTU-ArtFest22/facebook-chat-api/blob/795bdbc83323c98f546fa8084ed8b7eda10f100f/src/getThreadList.js#L15) to add backwards compatibility and setting `pending` to **false**.
```javascript
if(!callback && typeof pending === 'function') {
  callback = pending;
  pending = false; // Default pending to False
}
```

If you don't **everyone** has to update their scripts because of the extra parameter.